functions/public/styles/index.css

```diff
@@ -46,36 +46,85 @@ body{
     opacity: 0.8;
     text-align: right;
     z-index: 99999;
-    background-size: 400% 400%;
-	-webkit-animation: Gradient 60s ease infinite;
-    -moz-animation: Gradient 60s ease infinite;
-	animation: Gradient 60s ease infinite;
 }
 .left-cover{
-    margin: 200px 350px 0 250px;
+    top: 0;
+    left: 15;
     background: rgb(240, 240, 240);
-    color: black;
-    width: 30%;
+    color: #000000;
+    transition: all ease-in-out 400ms;
+    width: 400px;
     position: absolute;
-    top: 0;
     text-align: center;
-    height: 65%;
+    height: 100%;
+    line-height: 1.8em;
+}
+.leftstrip{
+    position: absolute;
+    padding: 0;
+    width: 15px;
+    height: 100%;
+    background: linear-gradient(to top, rgb(165, 24, 83), rgb(4, 45, 95));
+    opacity: 0.8;
+    background-size: 400% 400%;
+	-webkit-animation: Gradient 60s ease infinite;
+	-moz-animation: Gradient 60s ease infinite;
+	animation: Gradient 60s ease infinite;
+    top: 0;
+    left: 0;
 }
 #left-contain{
     margin: 0 auto;
     font-family: 'Quicksand Light', sans-serif;
     text-align: left;
     width: 70%;
+    transition: all ease-in-out 400ms;
     vertical-align: middle;
     font-size: 18.5px;
-    margin-top: 50px;
+    margin-top: 185px;
 }
 #left-contain h4{
     font-family: 'Pacifico', sans-serif;
     font-size:30px;
 }
+
+@-webkit-keyframes Gradient {
+	0% {
+		background-position: 0% 50%
+    }
```

Comment:
The closing curly bracket of this line is not the same as the closing culry bracket for the lines below (for 50% and 100%). could you please choose one consistent format and follow that? You could find follow Google's css style guide: https://google.github.io/styleguide/htmlcssguide.html (adding this to coding doc)

Reply:
Typing error there